src/elastio-snap.c

```diff
@@ -1246,6 +1255,26 @@ static inline int wrap_err_io(struct snap_device *dev){
     return !dev->sd_ignore_snap_errors ? -EIO : 0;
 }
 
+static void test_rw(struct snap_device *dev);
+
+static int param_set_write(const char *val, const struct kernel_param *kp)
+{
+	struct snap_device *dev;
+
+	dev = snap_devices[0];
+	test_rw(dev);
+
+	LOG_DEBUG("done.");
+	return 0;
+}
```

Comment:
It seems to be was necessary for testing during development and now can be removed. Correct?